docs/quickstart.md

```diff
@@ -79,6 +79,9 @@ Update the following variables specific to OCP.
  * `install_playbook_tag` : (Optional) [ocp4-playbooks](https://github.com/ocp-power-automation/ocp4-playbooks) ansible playbooks version to checkout.
  * `pull_secret_file` : (Optional) Location of the pull-secret file to be used. Default path is 'data/pull-secret.txt'.
  * `dns_forwarders` : (Optional) External DNS servers to forward DNS queries that cannot resolve locally. Eg: `"8.8.8.8; 9.9.9.9"`.
+ * `upgrade_image` : (Optional) Openshift release image having higher and supported version. If set, OCP cluster will be upgraded to this image version. (e.g. `"quay.io/openshift-release-dev/ocp-release-nightly@sha256:552ed19a988fff336712a50..."`)
```

Comment:
Perhaps a different section for Upgrade variables. Mention it support upgrading existing cluster created by the TF automation just by updating variables...

Reply:
Yep, let's use a different section so that it's not overwhelming. Or why don't we create a separate guide for upgrade and take it there ?  
We can keep a master list of variables with explanation but show the relevant ones in usage specific guides ?

Reply:
@bpradipt I have added a different section for the upgrade. ( the doc format is now inline with what we have fro PowerVM)